client/src/components/Contents/contentItem.js

```diff
@@ -0,0 +1,30 @@
+import Component from "../../core/component.js";
+import Item from "./item.js";
+
+export default class ContentItem extends Component{
+    template(){
+        return `
+            <li class="contentItem">
+                <h2 class="categoryName">${this.$props.key}</h2>
+                <div class="category">
+                    ${this.$state && Object.keys(this.$state).map(()=>`<Item data-component="Item"></Item>`).join('')}
+                </div>
+            </li>
+        `
```

Comment:
```suggestion
        const { $props, $state } = this;
        return `
            <li class="contentItem">
                <h2 class="categoryName">${$props.key}</h2>
                <div class="category">
                    ${$state && Object.keys($state).map(()=>`<Item data-component="Item"></Item>`).join('')}
                </div>
            </li>
        `
```
이렇게 중복코드를 줄여보면 어떨까요?
그리고 $state 자체에 배열이 들어가기보단 state property에 배열값을 선언해주는게 다른 사용자 입장에서 덜 헷갈릴 것 같습니다.